Makefile

```diff
@@ -19,7 +19,9 @@ docker-build:
 	docker build -t ursa -f ./Dockerfile .
 
 docker-run:
-	docker run -p 4069:4069 -p 4070:4070 -p 6009:6009 -p 8070:8070 --name ursa-cli -it ursa
+	docker run --name ursa-cli \
```

Comment:
we should add that to the docker file instead

Reply:
How so? 
- VOLUME in dockerfile only saves a volume generated on build, so it's not the same as passing flag
- we only have EXPOSE in dockerfile, if used ports should still be open to host for custom nginx setups etc right?

Reply:
Why would you want it updated in real time? What if a user doesn’t want that? Command should not be explicit to serve all needs. I would also remove the ports there

Reply:
Why save the ursa dir?
- Pem file for the nodes - this be a bigger deal with consensus
- cached content is supposed to be held for 30 days - again bigger deal once nodes have the hard requirement

If a user restarts the node without persistence, they get a completely new environment from scratch

Reply:
We don't have consensus

Reply:
Also I suggest reading adding volumes to docker file https://docs.docker.com/storage/volumes/

Reply:
okay, what about the existing docker-compose? This is just making the `make docker-run` exactly the same as `make compose-up`

Reply:
Docker compose is the use case, the container is the tool. Not intended way to run a prod ready node. Still for persistence we don't need the explicit command there read above linked

Reply:
Our documentation needs to be updated then if that's the case; most of the "getting started with docker" blog is about running the dockerfile or our published image directly, and only a mention at the end about the compose setup

Reply:
nonissue, should we close?

Reply:
I still think theres still a problem with not keeping it persistent. Because some users are updating their nodes hourly and running under new peer ids each time. For example, on the tracker there is one node that's registered with 33 different peer ids today

Reply:
Yeah let's update blogs stating to use docker compose or the one you mentioned. 

On the registration, peer ids can change the tls certificate in the future is far the only mapping that can't change